<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckoutCommitDialog</class>
 <widget class="QDialog" name="CheckoutCommitDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>161</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Check out commit</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>How do you want to check out this commit?</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QRadioButton" name="switchToLocalBranchRadioButton">
       <property name="text">
        <string>Switch to a &amp;branch that points here:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="switchToLocalBranchComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="detachedHeadRadioButton">
     <property name="text">
      <string>Enter &amp;detached HEAD here</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="createBranchRadioButton">
     <property name="text">
      <string>Start &amp;new branch here...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CheckoutCommitDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CheckoutCommitDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>switchToLocalBranchRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>switchToLocalBranchComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
